tests/unit/models/identifier-test.js

```diff
@@ -187,6 +187,28 @@ module('Unit | Model | identifier', function (hooks) {
         assert.true(isValid);
       });
 
+      test('it does not validate when localId was not changed', async function (assert) {
+        const structuredIdentifier = this.store().createRecord(
+          'structured-identifier',
+          {
+            localId: '123',
```

Comment:
**Issue: ** I suppose the test will pass even with a `changedAttributes`. Can you add some letters in the `localId` value? ;)